app/src/main/java/com/javg/cryptocurrencies/config/CRYConstantEndPoint.kt

```diff
@@ -1,7 +1,8 @@
 package com.javg.cryptocurrencies.config
 
```

Comment:
Since these constants are only used in CRYApi they could live in a companion object

Reply:
should my object live inside CRYApi?